host/usd/src/bin/usd-runner.rs

```diff
@@ -98,6 +98,10 @@ fn redirect() -> Result<(), anyhow::Error> {
     const VID: u16 = 0x0403; // Future Technology Devices International, Ltd
     const PID: u16 = 0x6011; // FT4232H Quad HS USB-UART/FIFO IC
     const DEVNO: usize = 2; // device #3 is the one we want
+    // macOS' FTDI driver does NOT support 4 Mbps so use a lower speed
+    #[cfg(target_os = "macos")]
+    const BAUD_RATE: u32 = 128_000;
```

Comment:
This method is fine but per the recent changes to #65 this should be turned up to `3_000_000`

Reply:
Rebased and updated to use 3 Mbps on macOS